backends/gdx-backend-lwjgl3/src/com/badlogic/gdx/backends/lwjgl3/Lwjgl3Graphics.java

```diff
@@ -513,6 +513,14 @@ public void setVSync (boolean vsync) {
 	@Override
 	public void setForegroundFPS (int fps) {
 		getWindow().getConfig().foregroundFPS = fps;
+		if (getWindow().getConfig().backgroundFPS < 0) {
+			getWindow().getConfig().backgroundFPS = fps;
+		}
```

Comment:
Is there a reason for this? The old LWJGL 2 backend didn't have this, see [LwjglGraphics.java#L681-L683](https://github.com/libgdx/libgdx/blob/e063c6dc2cdc078de30c97f7d7aedd574b5efc35/backends/gdx-backend-lwjgl/src/com/badlogic/gdx/backends/lwjgl/LwjglGraphics.java#L681-L683).